README.md

```diff
@@ -232,6 +232,13 @@ This script is executed when the Postgres container is launched.
 
 ***Note**: The updated `dump.pgdata` should be committed with the other associated changes.
 
+### **Running Single / Single-file Unit Tests**
+
+This step assumes that the user has installed VSCode, the Go language extension available [here](https://marketplace.visualstudio.com/items?itemName=golang.Go), and has successfully imported test data to their local database. 
+
+To run tests from within VSCode:
+In a FILENAME_test.go file, there will be a green arrow to the left of the method name, and clicking this arrow will run a single test locally. Tests should not be dependent upon other tests, but if a known-good test is failing, the user can run all tests in a given file by going to View -> Command Palette -> Go: Test Package, which will run all tests in a given file. 
+
```

Comment:
looks good! the only thing that we might want to consider adding is how `init()` can occasionally prevent single tests from running, depend on the test. It's not a great solution, but for now, we can comment out `init()` temporarily to get the test to run in isolation.

Reply:
Thank you! Added a comment on that

Reply:
perfect, thank you!